test/IRGen/condfail.sil

```diff
@@ -59,14 +66,16 @@ import Swift
 // CHECK-x86_64:      ud2
 // CHECK-i386:        ud2
 // CHECK-arm64:       brk
-// CHECK-armv7:       trap
-// CHECK-armv7s:      trap
+// CHECK-aarch64:     brk
+// CHECK-armv7:       [[TRAP]]
+// CHECK-armv7s:      [[TRAP]]
```

Comment:
I'm not sure what the value of the capture and match is (other than not having to duplicate the `.inst`.

Reply:
The only thing that I wanted to avoid is one instance using `trap`, and the next one using `.inst`. Saving the first one forces the second one to be the same. It is not a big deal (I don’t think LLVM will start randomly emitting one or the other), but avoids repeating the number four times.

Reply:
Its just odd since every other target doesn't do this.  I guess I care more about the targets being similar rather than the use of the match.